app/src/main/java/com/kks/nimblesurveyjetpackcompose/network/ApiInterface.kt

```diff
@@ -0,0 +1,12 @@
+package com.kks.nimblesurveyjetpackcompose.network
+
+import com.kks.nimblesurveyjetpackcompose.model.request.LoginRequest
+import com.kks.nimblesurveyjetpackcompose.model.response.BaseResponse
+import com.kks.nimblesurveyjetpackcompose.model.response.LoginResponse
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface ApiInterface {
```

Comment:
IMO, simply calling it `Api` is fine. API itself refers to an interface. 😄 

```suggestion
interface Api {
```